src/components/ContentUploader/ContentUploader.js

```diff
@@ -233,57 +236,64 @@ class ContentUploader extends Component<Props, State> {
      * @param {Array<UploadFileWithAPIOptions | File>} files
      */
     getNewFiles = (files: Array<UploadFileWithAPIOptions | File>) => {
+        const { rootFolderId } = this.props;
         const { itemIds } = this.state;
 
-        return Array.from(files).filter((file) => !(this.getFileId(file) in itemIds));
+        return Array.from(files).filter((file) => !(getFileId(file, rootFolderId) in itemIds));
```

Comment:
Not sure using "in" is the best. Isn't it outdated?
Better is to use array.includes

Reply:
`itemIds` is an object `{ [string]: boolean }`
I changed this to be just an object lookup instead, `!itemIds[getFileId(file, rootFolderId)]`